homeassistant/helpers/service.py

```diff
@@ -109,3 +110,41 @@ def extract_entity_ids(hass, service_call, expand_group=True):
             return [service_ent_id]
 
         return service_ent_id
+
+
+@callback
+def async_get_state_services(hass, domain, state):
+    """Return a dict of services for domain that can be used to set state.
```

Comment:
It is today already required to call multiple services to achieve some states. Eg. to have a cover to have a certain position and a certain tilt position it's required to call two different services.